.gitignore

```diff
@@ -1 +1,86 @@
 /vendor/
+composer.lock
+### JetBrains template
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio, WebStorm and Rider
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+# User-specific stuff
+.idea/**/workspace.xml
```

Comment:
I've added the entire `.idea` folder to this project's `.gitignore` file. This project shouldn't include editor specific config files.

Reply:
this config explicit includes some files that make sure all IntelliJ users have the same confiuration for the projekt. This files are just projekt-level configuration

Reply:
This project uses [EditorConfig](https://editorconfig.org/) an IDE/editor agnostic configuration system for open source contributions. It'd be best to get the PHPStorm/etc plugin installed for editor config instead of adding editor specific configs to this repo

Reply:
ok we will figure it out